mysqld_exporter.go

```diff
@@ -571,6 +624,93 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
 			}
 		}
 	}
+
+	if *perfEventsStatements {
+		// Timers here are returned in picoseconds.
+		perfSchemaEventsStatementsRows, err := db.Query(`
+SELECT
```

Comment:
I'd also indent this three tabs so it sits deeper than the `db.Query` surrounding it.